test/installer/scripts/omsadmin_systest.rb

```diff
@@ -296,4 +310,33 @@ def test_reconstruct_from_partial_empty_omsadmin_conf
                  "tmp directory was mistakenly created")
   end
 
+  def test_show_omsagent_proc_limit_default
+    set_omsagent_proc_limit()
+    lso = show_omsagent_proc_limit()
+    assert_match(/^-?\d+$/, lso, "The command to show what should be the default proc limit did not yield an integer.")
+  end
+
+  def test_show_omsagent_proc_limit_5000
+    set_omsagent_proc_limit(5000)
+    lso = show_omsagent_proc_limit()
+    nostr = lso.chomp
+    assert_equal("5000", nostr, "Should have been 5000 process limit.")
+  end
+
+  def test_show_omsagent_proc_limit_two_entries
+    FileUtils.cp("#{@omsadmin_test_dir}/limits-two-settings.conf", @proc_limits_conf)
+    lso = show_omsagent_proc_limit()
+    linearray = lso.split("\n")
+    l = linearray.length
+    v = linearray[l-1].chomp
+    assert_equal(6, l, "This example should have six output lines from stdout.")
+    assert_equal("100", v, "Multiple entries should still report last entry, in compliance with specification.")
+  end
+
+  def test_show_omsagent_proc_limit_no_entry
+    FileUtils.cp("#{@omsadmin_test_dir}/limits-no-settings.conf", @proc_limits_conf)
+    lso = show_omsagent_proc_limit()
+    assert_match(/info/, lso, "The integer associated with unlimited is -1, but we return only logging.")
```

Comment:
You should also check that "error" does not match in the output.

Reply:
Yes, I already changed that out just a bit ago.  I'll add the error check.  Thanks.

Reply:
Please see that I added return of "No Limit" rather than nothing, because I not only found it looked puzzling and assymetric, but I felt it might actually be confusing to not have a parallel value.  You're final call of course.